### PR TITLE
Updated readme

### LINKED ISSUE #428: Add documentation for non technical users
The readme for the project should cover the following:
 
- [x] What this demo is and why we created it
- [x] What technologies are used and why
- [x] simple user documentation (see #424)

### PR DESCRIPTION
**Pull request by ColinEberhardt**
Closes #428

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,15 +39,9 @@ npm install -g grunt-cli
 npm install
 ```
 
-- Perform an initial build:
```
Comment (ColinEberhardt) [inline README.md]: I noticed that `grunt build` no longer exists. Can someone please confirm the updated instructions? 

```
npm i
grunt serve
```

Comment (owennw) [inline README.md]: That looks right. The default grunt task is also `serve` so you don't need to mention it explicitly.

Comment (WPFerg) [inline README.md]: Correct.

**Comment by owennw** [comment]
Looks fine -- however this PR is going straight into master. Did you mean that?

**Comment by ColinEberhardt** [comment]
> Looks fine -- however this PR is going straight into master. Did you mean that?

I wasn't sure whether we had commits on dev that were not ready for release, so thought I'd pull direct to master for this change.

**Pull request merged**